src/DynamoSandbox/DynamoSandbox.csproj

```diff
@@ -98,6 +98,7 @@
     <ProjectReference Include="..\DynamoUtilities\DynamoUtilities.csproj">
       <Project>{B5F435CB-0D8A-40B1-A4F7-5ECB3CE792A9}</Project>
       <Name>DynamoUtilities</Name>
+      <Private>False</Private>
```

Comment:
what does this have to do with this PR? - and same below and for the other csproj at the top of these changes.